File: code/basic/controlStatement/ControlStatement.java
package code.basic.controlStatement;

public class ControlStatement {
    public static void main(String args[]) {
        int age = 18;
        if (age > 17) {
            System.out.println("Valid voter");
        } else {
            System.out.println("InValid voter");
        }
        int day = 1;
        switch (day) {
            case 1:
                System.out.println("Case 1");
                break;
            case 2:
                System.out.println("Case 2");
                break;

            default:
                System.out.println("Default");
                break;
        }
        int count = 0;
        while (count < 5) {
            System.out.println("count" + count);
            count++;
        }
        int count2 = 0;
        do {
            System.out.println("count" + count2);
            count2++;
        } while (count2 < 5);
    }
}

File: code/basic/controlStatement/controlStatement.py
def controlStatement():
    age = 18
    if age > 17:
        print("Voter valid")
    else:
        print("Voter invalid")
    count = 0
    while count < 5:
        print(count)
        count += 1
    for x in range(0, 5):
        print(x)

    def switchImplementation(params):
        dictionary = {
            'a': "A Called",
            'b': "B Called",
            'c': "C Called",
        }
        return dictionary.get(params)
    print(switchImplementation('a'))


controlStatement()
